stable/sumologic-fluentd/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: sumologic-fluentd
+version: 0.1.0
+description: Sumologic Log Collector
+keywords:
+  - monitoring
+  - logging
+sources:
+  - https://github.com/SumoLogic/fluentd-kubernetes-sumologic
+maintainers:
+  - name: Philip Champon
```

Comment:
Can you change this to `flah00`? We're starting to use GitHub usernames here instead.